src/SolutionTemplate/UnoSolutionTemplate.WinUI.netcore/Views/UnoQuickStart.csproj

```diff
@@ -2,10 +2,17 @@
 	<PropertyGroup>
 		<TargetFrameworks>$libarybasetargetframework$</TargetFrameworks>
 		$if$($UseWinUI$ == True)<TargetFrameworks Condition="$([MSBuild]::IsOSPlatform('Windows')) or '$(EnableWindowsTargeting)' == 'true'">$(TargetFrameworks);$basetargetframework$-windows10.0.18362</TargetFrameworks>$endif$
-		$if$($UseAndroid$ == True)<TargetFrameworks>$(TargetFrameworks);$basetargetframework$-android</TargetFrameworks>$endif$
-		$if$($UseIOS$ == True)<TargetFrameworks>$(TargetFrameworks);$basetargetframework$-ios</TargetFrameworks>$endif$
-		$if$($UseCatalyst$ == True)<TargetFrameworks>$(TargetFrameworks);$basetargetframework$-maccatalyst</TargetFrameworks>$endif$
-		$if$($UseAppKit$ == True)<TargetFrameworks>$(TargetFrameworks);$basetargetframework$-macos</TargetFrameworks>$endif$
+
+		<!-- 
+			This condition is used when building under Linux. If you're not building under
+			 Linux, you can remove this line and the conditions below.
+		-->
+		<_CanUseMobileTargets Condition="$([MSBuild]::IsOSPlatform('Windows')) or $([MSBuild]::IsOSPlatform('MacOS'))">true</_CanUseMobileTargets>
```

Comment:
I think you can use `"$([MSBuild]::IsOSPlatform('Linux'))`

Reply:
True, I originally put this because FreeBSD is in the list, but I don't think we'll be supporting that OS any time soon :)